devimint/src/federation.rs

```diff
@@ -170,17 +170,32 @@ impl Federation {
 
         run_dkg(admin_clients, params).await?;
 
-        let out_dir = &vars[&0].FM_DATA_DIR;
-        let cfg_dir = &process_mgr.globals.FM_CLIENT_DIR;
-        let out_dir = utf8(out_dir);
-        let cfg_dir = utf8(cfg_dir);
         // move configs to config directory
-        tokio::fs::rename(
-            format!("{out_dir}/invite-code"),
-            format!("{cfg_dir}/invite-code"),
+        let cfg_dir = utf8(&process_mgr.globals.FM_CLIENT_DIR);
+        let invite_code_filename_original = "invite-code";
+
+        // copy over invite-code file to client directory
+        let out_dir = utf8(&vars[&0].FM_DATA_DIR);
```

Comment:
It's not something this change introduces but `vars` is confusing and not informative. :D